spec/system/profile_spec.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 RSpec.describe "profile page" do
-  it "allows the user to edit their username" do
+  before do
```

Comment:
We've been moving in the direction of eliminating `before` calls. I think it's fine to copy these over to these lines to the new spec.

Reply:
Your Rubocop config didn't allow me too, it was too many lines for what I needed. By the way I spent pretty much half the time fighting with Rubocop, I would recommend just using standardrb. I can imagine being too strict here can actually drive away contributors ;)

Reply:
Hmm, it works for me when I move it down. It's limited to 5 lines, and you've only got 3.

> I can imagine being too strict here can actually drive away contributors

Not having strict linters would drive _me_ away, and I'm [one of the most active contributor this project has ever had](https://github.com/stringer-rss/stringer/graphs/contributors). I'm happy to help others figure out how to gracefully appease them, though.

I'm not a fan of libraries like standardrb. They're not strict enough for my taste and not flexible enough.